src/line.js

```diff
@@ -396,6 +394,11 @@ define([
       }
     }
 
+    // If width changed when we don't want
```

Comment:
When is it possible? =)

Reply:
It can happen when going from time to pixels as we must use Math.round. So the width sometimes gains/loses 1 pixel. Because of that, if we always update the width, the source is "shaking".

Reply:
Could we use Math.floor or Math.ceil, instead?

Reply:
Also, could you add the explanation to the code as a comment, please?
It would be better to keep it there instead of the current comment.

Reply:
The problem of approximation will remain I guess. I will detail a bit more the comment in the code anyway.